### ISSUE TITLE
Changing the format / indent does not seem to work

### ISSUE THREAD

**Issue by biergit** [jhipster/jhipster-lite#4199]
I am using the UI provided by the Docker container. I only get indent size of two spaces.
The log shows:
> t.j.l.p.i.s.TraceProjectFormatter        : No npm installed, can't format project

**Comment by DamnClin** [comment]
I guess you are clicking the format button somewhere in the process? We need to update this button activation to be bound to prettier application

**Comment by biergit** [comment]
Yes I clicked "Apply selected" only then the "Format" button became active and I clicked it. Also I noticed changing the line ending has no effect - I always get "lf".

**Comment by DamnClin** [comment]
Did you select prettier module?

**Comment by biergit** [comment]
I didn't but I checked now and nothing changed.

**Comment by DamnClin** [comment]
Just made some test, forgot the activation updates were already done (here https://github.com/jhipster/jhipster-lite/pull/3545)...

Ok, looks like you don't have npm installed on your computer so the formatting can be done.

**Comment by biergit** [comment]
I used the Docker container. And I see that the [TraceProjectFormatter](https://github.com/jhipster/jhipster-lite/blob/fd8f131e453c47491889456bf19215629a87ea59/src/main/java/tech/jhipster/lite/project/infrastructure/secondary/TraceProjectFormatter.java) is not trying too hard to even check for NPM :)

**Comment by DamnClin** [comment]
Yep, this the instance used when no npm installation can be found :D. We need to add npm in the published docker image :)

**Comment by pascalgrimaud** [comment]
Indeed, it would be a good enhancement

**Comment by DamnClin** [comment]
@pascalgrimaud where is the root image used to publish on docker hub?

**Comment by pascalgrimaud** [comment]
@DamnClin : it comes from https://github.com/jhipster/jhipster-lite/blob/main/Dockerfile

**Comment by DamnClin** [comment]
Ok, adding npm should be pretty straightforward so, do you consider it a good first issue?

**Comment by pascalgrimaud** [comment]
I don't think it's a good first issue :-D
Not so easy to find the good solution

**Comment by DamnClin** [comment]
Ok, my bad, I though this just needs something like this: 

```
ARG NVM_VERSION=v0.39.2
ARG NODE_VERSION=v18.12.1
ENV NVM_DIR=/root/.nvm

RUN curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/${NVM_VERSION}/install.sh | bash \
    && . $NVM_DIR/nvm.sh \
    && nvm install $NODE_VERSION \
    && nvm alias default $NODE_VERSION \
    && nvm install-latest-npm \
    && nvm use default
ENV PATH $NVM_DIR/versions/node/$NODE_VERSION/bin:$PATH
```

**Comment by pascalgrimaud** [comment]
Maybe your solution can work, but it needs to be tested

**Comment by DamnClin** [comment]
Of course, just dropped some stuff here
